src/libs/API/parameters/CreateAppReportParams.ts

```diff
@@ -1,5 +1,6 @@
 type CreateAppReportParams = {
-    reportName: string;
+    // Pass reportName only if it's a custom user defined name
```

Comment:
NAB: We should remove this param entirely because it's never used. We don't allow you to specify the name when calling this function, it always uses the default or the name according to the formula.